libp2p/NodeTable.cpp

```diff
@@ -413,21 +414,28 @@ void NodeTable::onReceived(UDPSocketFace*, bi::udp::endpoint const& _from, bytes
 				auto in = dynamic_cast<Pong const&>(*packet);
 				// whenever a pong is received, check if it's in m_evictions
 				bool found = false;
+				NodeID leastSeenID;
 				EvictionTimeout evictionEntry;
 				DEV_GUARDED(x_evictions)
-					for (auto it = m_evictions.begin(); it != m_evictions.end(); ++it)
-						if (it->first.first == in.sourceid && it->first.second > std::chrono::steady_clock::now())
-						{
-							found = true;
-							evictionEntry = *it;
-							m_evictions.erase(it);
-							break;
+					{ 
```

Comment:
Please make it one level less indented (see in other places in this file braces directly after `DEV_GUARDED` are the same level of indentation as `DEV_GUARDED`)